src/apps/loyalty/server/index.tsx

```diff
@@ -69,6 +70,18 @@ app.get("/inquiries", (req, res) => {
     return res.redirect(req.baseUrl + "/login")
   }
 
+  const headers = { "X-Access-Token": req.user.get("accessToken") }
```

Comment:
Actually, the endpoint is separate so that's fine. we can just wrap it in a method on the CurrentUser backbone model though that we have access to that data on all the routes.

There are other routes (e.g. `/thank-you`) where we need to check if the user has already submitted inquiries or if it's a confirmed buyer